context/app/static/js/pages/Genes/Genes.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import TableOfContents from 'js/shared-styles/sections/TableOfContents';
+import { getSections } from 'js/shared-styles/sections/TableOfContents/utils';
+import Summary from 'js/components/genes/Summary';
+import { FlexRow, Content } from './style';
+
+function Genes() {
+  const summaryId = 'Summary';
+  const hraId = 'Human Reference Atlas';
+  const referenceId = 'Reference-Based Analysis';
+  const assaysId = 'Assays';
+  const samplesId = 'Samples';
```

Comment:
These constants are being reinstantiated on every render and should be moved outside of the component:

```suggestion
const summaryId = 'Summary';
const hraId = 'Human Reference Atlas';
const referenceId = 'Reference-Based Analysis';
const assaysId = 'Assays';
const samplesId = 'Samples';

function Genes() {
```